### ISSUE TITLE
midas minigame

### ISSUE THREAD

**Issue by Geczy** [dotabod/backend#95]
add efficiency to midas and in the fallowing of the value in the chat the singing text

![image](https://user-images.githubusercontent.com/1036968/211892769-f7258e01-693e-4425-aefb-f40734cdb2c5.png)

![image](https://user-images.githubusercontent.com/1036968/211892805-09302ae6-6068-4f48-8c40-dfa6bcd19cf7.png)

https://discord.com/channels/1039887907705593876/1046119849014853775/1062583187492065290

**Comment by dankYoff** [comment]
maybe add a separate command !midas

**Comment by vsmelnikov** [comment]
it will look better after game end, like mini statistic.

**Comment by dankYoff** [comment]
![зображення](https://user-images.githubusercontent.com/91318807/212052451-67ff6e0a-0671-462f-9f7d-51ae4c2c8ec6.png)
this is the new midas formula, the old formula incorrectly calculates the total time that has passed since the purchase of midas

**Comment by dankYoff** [comment]
certain problems begin here: 1) is it possible to track Midas, who is in the illusion of Arkwarden and Londruid?
2) is it possible to track the time of purchase of midas and the current time

**Comment by dankYoff** [comment]
> it will look better after game end, like mini statistic.
yes i agree with you, it's better to do it at the end of the match than to spam every time the efficiency will hardly change unless you bought an octarin or play arc/lone druid

**Comment by dankYoff** [comment]
of course, it is better to keep previous statistics of not pressing midas like 30s+
and at the end display detailed statistics like this

> Efficiency Midas - 68%. Number of uses: 5/8 in 25 minutes
